yt_dlp/extractor/ximalaya.py

```diff
@@ -82,6 +114,24 @@ def _real_extract(self, url):
             audio_info_file, audio_id,
             f'Downloading info json {audio_info_file}', 'Unable to download info file')
```

Comment:
I know this was here before, but can you please simplify this using:
```suggestion
            f'{scheme}://m.ximalaya.com/tracks/{audio_id}.json', audio_id,
            'Downloading info json', 'Unable to download info file')
```

Reply:
got it